compliance_checker/cf/util.py

```diff
@@ -5,7 +5,11 @@
 
 import requests
 from cf_units import Unit
-from importlib_resources import files
+try:
+    from importlib.resources import files
+except ImportError:
+    from importlib_resources import files
```

Comment:
This is not necessary b/c importlib_resources is installable in Python >=3.9 and, as far as I know, it is just a stub for the std `importlib` there.

Reply:
This was intended as future proofing -- this way, when we drop 3.8, all we need to do is remove the dependency, rather than going in and changing all the imports later -- and it allows users with never Python versions to run without importlib_resources.

It's a matter of taste -- do with it what you will

Reply:
We would need to remove that if-clause to reduce the complexity too. In a way, `importlib_resources` is maintained and supported for py>3.9, so we don't need to change it as soon as we drop 3.8. It is a minor maintenance chore.